src/pages/Systems/DeleteIngestorModal.tsx

```diff
@@ -0,0 +1,57 @@
+import { Stack, Text, Group, Button, Modal } from '@mantine/core';
+import { useDeleteIngestor } from '@/hooks/useClusterInfo';
+import classes from './styles/Systems.module.css';
+import { useCallback } from 'react';
+
+function sanitizeIngestorUrl(url: string) {
+	if (url.startsWith('http://')) {
+		url = url.slice(7);
+	} else if (url.startsWith('https://')) {
+		url = url.slice(8);
+	}
+
+	if (url.endsWith('/')) {
+		url = url.slice(0, -1);
+	}
+
+	return url;
+}
+
+const ModalTitle = () => {
+	return <Text style={{ fontWeight: 600, marginLeft: '0.5rem' }}>Confirm Delete</Text>;
```

Comment:
I don't think the indentation spaces used here are correct. 
Please follow the existing configuration.